client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -101,6 +101,7 @@ export class ProductPurchaseFeaturesList extends Component {
 				/>
 				<CustomDomain selectedSite={ selectedSite } hasDomainCredit={ planHasDomainCredit } />
 				<BusinessOnboarding
+					isWPcomPlan
```

Comment:
Nit: Although WP is an abbreviation, `isWPcomPlan` should be `isWpcomPlan` if we follow our naming conventions. That's easy to confirm if you search for `WPcom` and `Wpcom` in the calypso codebase.

Reply:
I noticed the same, but for some reasons I decided to go for `WPcomPlan`. 🤦🏽‍♂️Updated. 👍🏽